src/rust-2018/error-handling-and-panics/the-question-mark-operator-for-easier-error-handling.md

```diff
@@ -120,131 +118,3 @@ case, `?` will return a value for `Some(T)` and return `None` for `None`. One
 current restriction is that you cannot use `?` for both in the same function,
 as the return type needs to match the type you use `?` on. In the future,
 this restriction will be lifed.
-
```

Comment:
sorry, why are we removing all of this?

Reply:
Isn't this for that ` ? in main and tests ` section?